sanic/app.py

```diff
@@ -214,9 +214,12 @@ def add_route(self, handler, uri, methods=frozenset({'GET'}), host=None,
         return handler
 
     # Decorator
-    def websocket(self, uri, host=None, strict_slashes=False):
+    def websocket(self, uri, subprotocols=None, host=None,
```

Comment:
Can we move subprotocols to be the last keyword-arg? I don't want to break anyone's existing functionality by adding a new positional arg in between the already existing ones.